### DISCUSSION TITLE
Support not responding to invalid school email address

### DISCUSSION THREAD

**Discussion by parker02311** [community/community#114516] [GitHub Education]
### Hi everyone,

this is like my 6th time trying to get this fixed. Our school appears to have the wrong information inputted for identification. When trying to use school provided email address I get the error `We require applicants to use a school-issued email address to apply.`, however when trying to use a gmail account it works fine. Even while the gmail account works fine my valid school ID gets automatically declined even though it provides full name (same as GitHub), the name of the school, and both the years 2023 and 2024 on the ID.

I have opened a GitHub support ticket but have not received a response in over a week since opening the ticket. If anyone has had similar issues and knows a way to fix that would be greatly appreciated!

**Comment by samus-aran** [comment]
Hi @parker02311 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
